docs/build-for-developers/cli-installation.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Using the CLI
+sidebar_label: Installation and Updating
+slug: /cli-installation
+---
+
+## Installation
+
+**To install**
+
+```bash
+npm install -g @openfn/cli
+```
+
+**Make sure everything works by running the built-in test job**
+
+```bash
+openfn test
+```
+
+The word `openfn` will invoke the CLI. The word `test` will invoke the test
+command.
+
+<details><summary>You should see some output like this:</summary>
+
+    [CLI] ℹ Versions:
+            ▸ node.js     18.12.1
+            ▸ cli         0.4.11
+            ▸ runtime     0.2.2
+            ▸ compiler    0.0.38
+    [CLI] ℹ Running test job...
+    [CLI] ℹ Workflow object:
+    [CLI] ℹ {
+        "start": "start",
+        "jobs": [
+        {
+            "id": "start",
+            "data": {
+            "defaultAnswer": 42
+            },
+            "expression": "const fn = () => (state) => { console.log('Starting computer...'); return state; }; fn()",
+            "next": {
+            "calculate": "!state.error"
+            }
+        },
+        {
+            "id": "calculate",
+            "expression": "const fn = () => (state) => { console.log('Calculating to life, the universe, and everything..'); return state }; fn()",
+            "next": {
+            "result": true
+            }
+        },
+        {
+            "id": "result",
+            "expression": "const fn = () => (state) => ({ data: { answer: state.data.answer || state.data.defaultAnswer } }); fn()"
+        }
+        ]
+    }
+
+    [CLI] ✔ Compilation complete
+    [R/T] ♦ Starting job start
+    [JOB] ℹ Starting computer...
+    [R/T] ℹ Operation 1 complete in 0ms
+    [R/T] ✔ Completed job start in 1ms
+    [R/T] ♦ Starting job calculate
+    [JOB] ℹ Calculating to life, the universe, and everything..
+    [R/T] ℹ Operation 1 complete in 0ms
+    [R/T] ✔ Completed job calculate in 1ms
+    [R/T] ♦ Starting job result
+    [R/T] ℹ Operation 1 complete in 0ms
+    [R/T] ✔ Completed job result in 0ms
+    [CLI] ✔ Result: 42
+
+</details>
+
+What we've just done is executed a JavaScript expression, which we call a _job_.
+The output prefixed with `[JOB]` comes directly from `console.log` statements in
+our job code. All other output is the CLI trying to tell us what it is doing.
+
+**Check the version**
+
+```bash
+openfn -v
+```
+
+**Get help**
+
+```bash
+openfn help
+```
+
+## Updating
+
+**You should be able to install a new version straight on top of your current
```

Comment:
Be confident: You CAN install a new version straight on top.

I would like to leave the "if it doesn't work" stuff to a troubleshooting page somewhere, because I don't think that will happen often? (Nothing to do with my code - that's all about npm). But I'm open to other opinions on this.